### PR TITLE
Db 4620 improve syntax person rol organisation attributes

### PR DESCRIPTION
**Pull request by MiguelAHM**

### CODE REVIEW

File: whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java [hunk 1]
```diff
@@ -400,9 +400,13 @@ public interface AttributeSyntax extends Documented {
 
     AttributeSyntax ORG_NAME_SYNTAX = new AttributeSyntaxRegexp(
             Pattern.compile("(?i)^[\\]\\[A-Z0-9._\"*()@,&:!'`+\\/-]{1,64}( [\\]\\[A-Z0-9._\"*()@,&:!'`+\\/-]{1,64}){0,29}$"), "" +
-            "A list of 1 to 30 words separated by white space. A word is made up of letters, digits and the following characters:\n" +
+            "A list of 1 to 30 words separated by white space\n." +
+            "It just allow ASCII alphanumeric characters.\n" +
```
Comment (eshryane) [inline whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java:404]: I suggest `A word may only contain ASCII characters.` Not just alphanumeric - punctuation is also allowed.

File: whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java [hunk 2]
```diff
@@ -898,11 +901,12 @@ public boolean matches(final ObjectType objectType, final String value) {
         public String getDescription(final ObjectType objectType) {
             return "" +
                     "It should contain 2 to 10 words.\n" +
-                    "Each word consists of letters, digits or the following symbols:\n" +
+                    "A word is made up of ASCII characters and additionally:\n" +
```
Comment (eshryane) [inline whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java:904]: be more precise, `A word is made up of ASCII alphanumeric characters....`, as not all `ASCII characters` are allowed.

File: whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java [hunk 3]
```diff
@@ -898,11 +901,12 @@ public boolean matches(final ObjectType objectType, final String value) {
...
                     ".`'_-\n" +
                     "The first word should begin with a letter.\n" +
                     "At least one other word should also begin with a letter.\n" +
-                    "Max 64 characters can be used in each word.";
+                    "Max 64 characters can be used in each word.\n" +
+                    "If this don't comply your requirements, we suggest to use remarks or description attribute";
```
Comment (eshryane) [inline whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java:909]: I suggest not changing this text

File: whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java [hunk 4]
```diff
@@ -400,9 +400,12 @@ public interface AttributeSyntax extends Documented {
 
     AttributeSyntax ORG_NAME_SYNTAX = new AttributeSyntaxRegexp(
             Pattern.compile("(?i)^[\\]\\[A-Z0-9._\"*()@,&:!'`+\\/-]{1,64}( [\\]\\[A-Z0-9._\"*()@,&:!'`+\\/-]{1,64}){0,29}$"), "" +
-            "A list of 1 to 30 words separated by white space. A word is made up of letters, digits and the following characters:\n" +
+            "A list of 1 to 30 words separated by white space\n." +
+            "A word is made up of ASCII characters and additionally:\n" +
```
Comment (eshryane) [inline whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java:404]: more precisely, `ASCII alphanumeric characters`

File: whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java (hunk 3 above)
Comment (MiguelAHM) [reply whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java]: I removed this line from both then

**Review by eshryane — APPROVED** [review]

**Pull request merged**